.github/workflows/ci.yml

```diff
@@ -17,12 +17,7 @@ jobs:
         test-type: [lint, check-migrations, test]
     services:
       postgres:
-        # Heroku uses PostGIS 2.5.5 and PostgreSQL 13.5
-        # This is technically not a valid combination
-        # according to PostGIS and is therefore not
-        # not an image they offer. We use PostgreSQL 12.x
-        # and PostGIS 2.5.x instead.
-        image: postgis/postgis:12-2.5
+        image: postgis/postgis:14-3.2
```

Comment:
A note for others: when I tried out this branch, postgis would not start with the following error. 

```
2022-08-01 19:30:07.896 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 12, which is not compatible with this version 14.4 (Debian 14.4-1.pgdg110+1).
```

I performed a `docker-compose down` to clear the db container completely. Then You can start anew with `docker-compose up` and `python manage.py migrate && python manage.py populate` (within the django container as always)